src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -900,20 +964,36 @@ namespace winrt::TerminalApp::implementation
             // Wraparound math. By adding tabCount and then calculating modulo tabCount,
             // we clamp the values to the range [0, tabCount) while still supporting moving
             // leftward from 0 to tabCount - 1.
-            _SetFocusedTabIndex(((tabCount + *index + (bMoveRight ? 1 : -1)) % tabCount));
+            const auto newTabIndex = ((tabCount + *index + (bMoveRight ? 1 : -1)) % tabCount);
+            _SelectTab(newTabIndex);
```

Comment:
?

Reply:
Oh, see, `_SetFocusedTabIndex` will only do the dispatcher thing, while `_SelectTab` will to the thing _now_ if we're in startup, otherwise it'll do the `_SetFocusedTabIndex` thing. This basically just makes `focus-tab -n` and `focus-tab -p` work